cli/command/run/cmd.go

```diff
@@ -87,6 +89,8 @@ func Command(cli *command.SauceCtlCli) *cobra.Command {
 	cmd.Flags().StringVar(&testEnv, "test-env", "docker", "Specifies the environment in which the tests should run. Choice: docker|sauce.")
 	cmd.Flags().BoolVarP(&showConsoleLog, "show-console-log", "", false, "Shows suites console.log locally. By default console.log is only shown on failures.")
 	cmd.Flags().IntVar(&concurrency, "ccy", 1, "Concurrency specifies how many suites are run at the same time.")
+	cmd.Flags().StringVar(&tunnelID, "tunnel-id", "", "Overrides the sauce-connect tunnel-id to be used for the run.")
```

Comment:
I'd change the wording a little. It's not necessarily that this is meant to override what's in the config. That's just a consequence of specifying values via the CLI args. But the user is free to just specify those values via CLI alone and never have to define what's in the config.

```suggestion
	cmd.Flags().StringVar(&tunnelID, "tunnel-id", "", "Sets the sauce-connect tunnel ID to be used for the run.")
```